app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -272,9 +273,10 @@ export const ContentListView: React.FC = () => {
         </div>
         <div className="box">
           <h2 className="caps">Advanced Search</h2>
-          <div>
+          <div style={{ display: 'flex', flexDirection: 'row', minWidth: '550px' }}>
```

Comment:
Don't love inline styles

Reply:
Will resolve the above and other stylings in my styling ticket assigned for this sprint